README.md

```diff
@@ -113,6 +114,9 @@ use Microsoft\Kiota\Abstractions\ApiException;
 use Microsoft\Graph\Core\Authentication\GraphPhpLeagueAuthenticationProvider;
 use Microsoft\Kiota\Authentication\Oauth\ClientCredentialContext;
 
+set_include_path(DIR);
+require '../vendor/autoload.php';
+
```

Comment:
This is heavily dependent on the customer's project structure. I'd leave it out of the samples but mention it in a separate section in case the developer is facing import/PHP setup errors.

Should we create a separate Markdown file on Setting up your PHP environment and link to it? It can include the extensions required & fixes on imports in case the developer is new to PHP?

Reply:
@Ndiritu yes! That would be a better approach. In that document we can also mention the cacert file to the php.ini in the curl.cainfo = path to file. Let me start with something in this same PR.